isis/src/qisis/objs/BundleObservationView/AS15_16_test_bundleout_images.csv

```diff
@@ -1,8 +0,0 @@
-Image,rms,rms,rms,X,X,X,X,X,Y,Y,Y,Y,Y,Z,Z,Z,Z,Z,RA,RA,RA,RA,RA,DEC,DEC,DEC,DEC,DEC,TWIST,TWIST,TWIST,TWIST,TWIST,
```

Comment:
did you mean to check this in?

Reply:
This is will actually be removed when the PR is merged (It exists in dev, but not ipce).